### DISCUSSION TITLE
"opModeInInit" in RobotAutoDriveToLine_Linear

### DISCUSSION THREAD

**Discussion by ad25343** [FIRST-Tech-Challenge/FtcRobotController#331] [General] [answered]
Hello,
I am getting the following error after I pulled the new version of FtcRobotController. Not sure if I made mistake when I did a pull request or if something is missing. Any ideas? Thanks in advance for the help. 

error: cannot find symbol
        while (opModeInInit()) {
               ^
  symbol:   method opModeInInit()
  location: class RobotAutoDriveByGyro_Linear

Same error for RobotAutoDriveToLine_Linear

AD

**Comment by cmacfarl** [comment, marked as answer]
opModeInInit() is a method that was added in v7.2.

Can you describe how you upgraded to v7.2?

Did you have any conflicts in build.dependencies.gradle?  Can you post the contents of that file here?

**Reply by ad25343** [reply]
Hello Craig, Appreciate the quick response. I may have messed up my set-up when I tried to upgrade to 7.2. Doing a clean install now. 
We can close this thread. I will come back if I still run into issues. 

Regards,
AD

**Reply by ad25343** [reply]
Issue is resolved after clean install. Thank you.
